modules/microshift-install-multus-rpm.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-multus_{context}"]
+== Installing the multiple networks plugin
+
+Use this procedure to install the {microshift-short} Multus CNI plugin alongside a new {microshift-short} installation. The {microshift-short} Multus Container Network Interface (CNI) plugin is not installed by default. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package.
+
+[IMPORTANT]
+====
+The Multus RPM should not be removed after you install it.
```

Comment:
I'm confused by this mention of removing the RPM? To me "remove" means use the dnf command to uninstall the RPM, which is the opposite of what we are telling the user to do in this section. Maybe it's just a suggestion to remove the rpm file, as it would no longer be needed and just take up space; but that's just general system administrator best practice, so I don't know why we'd need to call it out here?

Reply:
We're warning users that this is an installation you do not want to reverse. So don't proceed unless you really want to use multus.

Reply:
Otherwise, does this PR look LGTM?